classes/classes/Scenes/NPCs/JojoScene.as

```diff
@@ -2307,11 +1984,11 @@ public function lowCorruptionJojoEncounter():void
 
 	outputText("Tired of exploring the forest for the moment, you decide to head back to camp.  Not feeling like taking the scenic route, you move to step through some bushes, but immediately your mind registers a yelp.  The instant you move to look at the source of the noise, a white blur smacks you right on your head.");
 
-	if (player.tou >= 50 && player.isBiped() == true)
+	if (player.tou >= 50 && player.isBiped())
 	{
 		outputText("  You take a few steps back, momentarily dazed.  Shaking it off, you ready your [weapon] and assume a fighting stance.\n\n");
 	}
-	else if (player.tou < 50 && player.isBiped() == false)
+	else if (player.tou < 50 && player.isBiped())
```

Comment:
That should then be `&& !player.isBiped())`

Reply:
Thanks for catching that.

Thinking about it some more, this piece of code seems wrong, as it does not make much sense in context with the rest of the if/else blocks.

Change as requested and then submit a new PR to fix this?

Reply:
The code was introduced in e8f9f0071b7fc8d7d87845a8db35431a04c18ffd.
No idea if this is working as intended.